lib/modules/manager/nuget/extract.ts

```diff
@@ -46,6 +48,20 @@ function extractDepsFromXml(xmlNode: XmlDocument): NugetPackageDependency[] {
     const child = todo.pop()!;
     const { name, attr } = child;
 
+    if (name === 'ContainerBaseImage') {
+      const dep = getDep(child.val, true);
+
+      if (is.nonEmptyStringAndNotWhitespace(dep.depName)) {
+        results.push({
+          datasource: DockerDatasource.id,
```

Comment:
needs also be added to supported datasource in index.ts

Reply:
Done.